`test_rrelu` test is flaky

This is part of https://github.com/chainer/chainer/issues/6903 effort.

- Failure example: https://ci.preferred.jp/chainer.py27and35/2279/
- Failure test: chainer_tests/functions_tests/activation_tests/test_rrelu.py
- Failure test case: `_ TestRReLU_param_3_{dtype=float16, shape=(), train=False}_use_chainerx_false__chainerx_device_None__use_cuda_true__cuda_device_1__use_cudnn_never__cudnn_deterministic_false__autotune_false__cudnn_fast_batch_normalization_false__use_ideep_never.test_backward _`

```
E           Not equal to tolerance rtol=0.005, atol=0.0005
E           
E           Mismatch: 100%
E           Max absolute difference: 0.00933533
E           Max relative difference: 0.01118056
E            x: array(-0.825626)
E            y: array(-0.834961)
```

--- a/tests/chainer_tests/functions_tests/activation_tests/test_rrelu.py
+++ b/tests/chainer_tests/functions_tests/activation_tests/test_rrelu.py
@@ -33,8 +33,6 @@ from chainer.testing import attr
 }))
 class TestRReLU(testing.FunctionTestCase):
 
-    dodge_nondifferentiable = True
-
     def setUp(self):
         # Assumption l < u
         self.l = numpy.random.uniform(0, 1)
@@ -52,6 +50,8 @@ class TestRReLU(testing.FunctionTestCase):
 
     def generate_inputs(self):
         x = numpy.random.uniform(-1, 1, self.shape).astype(self.dtype)
+        if self.test_name in ('test_backward', 'test_double_backward'):
+            x[(-0.05 < x) & (x < 0.05)] = 0.5
         return x,
 
     def forward(self, inputs, device):

